pkg/kubelet/kuberuntime/logs/logs.go

```diff
@@ -453,16 +449,13 @@ func waitLogs(ctx context.Context, id string, w *fsnotify.Watcher, runtimeServic
 			return false, false, fmt.Errorf("context cancelled")
 		case e := <-w.Events:
 			switch e.Op {
-			case fsnotify.Write:
+			case fsnotify.Write,
+				fsnotify.Rename,
+				fsnotify.Remove,
+				fsnotify.Chmod:
```

Comment:
nit: fsnotify.Write looks separated. Having all items in one line would be more readable.